packages/main/src/Breadcrumbs.js

```diff
@@ -0,0 +1,534 @@
+import BreadcrumbsDesign from "./types/BreadcrumbsDesign.js";
+import BreadcrumbsSeparatorStyle from "./types/BreadcrumbsSeparatorStyle.js";
+import BreadcrumbsItem from "./BreadcrumbsItem.js";
+import {
+	BREADCRUMBS_ARIA_LABEL,
+	BREADCRUMBS_OVERFLOW_ARIA_LABEL,
+	BREADCRUMBS_CANCEL_BUTTON,
+} from "./generated/i18n/i18n-defaults.js";
+import Link from "./Link.js";
+import Label from "./Label.js";
+import ResponsivePopover from "./ResponsivePopover.js";
+import Popover from "./Popover.js";
+import List from "./List.js";
+import StandardListItem from "./StandardListItem.js";
+import Icon from "./Icon.js";
+
+// Templates
+import BreadcrumbsTemplate from "./generated/templates/BreadcrumbsTemplate.lit.js";
+import BreadcrumbsPopoverTemplate from "./generated/templates/BreadcrumbsPopoverTemplate.lit.js";
+
+// Styles
+import breadcrumbsCss from "./generated/themes/Breadcrumbs.css.js";
+import breadcrumbsPopoverCss from "./generated/themes/BreadcrumbsPopover.css.js";
+
+/**
+ * @public
+ */
+const metadata = {
+	tag: "ui5-breadcrumbs",
+	managedSlots: true,
+	languageAware: true,
+	slots: /** @lends sap.ui.webcomponents.main.Breadcrumbs.prototype */ {
+
+		/**
+		 * Defines the component items.
+		 *
+		 * <br><br>
+		 * <b>Note:</b> Use the <code>ui5-breadcrumbs-item</code> component to define the desired items.
+		 * @type {sap.ui.webcomponents.main.IBreadcrumbItem[]}
+		 * @slot items
+		 * @public
+		 */
+		"default": {
+			propertyName: "items",
+			type: HTMLElement,
+			invalidateOnChildChange: true,
+		},
+
+	},
+	properties: /** @lends  sap.ui.webcomponents.main.Breadcrumbs.prototype */ {
+
+		/**
+		 * Defines the visual indication and behavior of the breadcrumbs.
+		 * Available options are <code>Standard</code> (by default) and <code>NoCurrentPage</code>.
+		 * <br><br>
+		 * <b>Note:</b> The <code>Standard</code> breadcrumbs show the current page as the last item in the trail.
+		 * The last item contains only plain text and not a link.
+		 *
+		 * @type {BreadcrumbsDesign}
+		 * @defaultvalue "Standard"
+		 * @public
+		*/
+		design: {
+			type: BreadcrumbsDesign,
+			defaultValue: BreadcrumbsDesign.Standard,
+		},
+
+		/**
+		 * Determines the visual style of the separator between the breadcrumb items.
+		 *
+		 * @type {BreadcrumbsSeparatorStyle}
+		 * @defaultvalue "Slash"
+		 * @public
+		 */
+		separatorStyle: {
+			type: BreadcrumbsSeparatorStyle,
+			defaultValue: BreadcrumbsSeparatorStyle.Slash,
+		},
+
+		/**
+		 * Holds the number of link-items in the overflow
+		 *
+		 * @type {integer}
+		 * @defaultvalue 0
+		 * @private
+		 */
+		_countItemsInOverflow: {
+			type: Integer,
+			noAttribute: true,
+			defaultValue: 0,
+		},
+
+	},
+	events: /** @lends  sap.ui.webcomponents.main.Breadcrumbs.prototype */ {
+
+		/**
+		 * Fired when a link is activated.
+		 *
+		 * @event sap.ui.webcomponents.main.Breadcrumbs#link-click
+		 * @param {HTMLElement} link The clicked link.
+		 * @public
+		 */
+		"link-click": {
```

Comment:
`item-click` might be also a candidate, as the child component  is BreadcrumbsItem and the Breadcrumbs slot is referred as "items" (although default), maybe you can consider that.

Reply:
yes, I pondered earlier about the event name!
The name "item-click" is consistent with the slot name, but I named it "link-click" because currently no event is fired **if the clicked item is the one that stands for the "current-location" label.** For now I see two options:

1) fire the event even when the  "current-location" label is pressed - but if we want to keep the label as non-interactive, then I discard this option
OR
2) rename the event from "link-click" to "item-click" nevertheless, for consistence with the slot name

What do you think?
Thnx!

Reply:
Eventually refactored to "item-click" that fires on click on *any* item, as it makes most sense